README.md

```diff
@@ -1 +1,118 @@
-# Java MultiPlatform Scenarios
\ No newline at end of file
+# Java MultiPlatform Scenarios <a href="https://www.browserstack.com/"><img src="https://www.vectorlogo.zone/logos/browserstack/browserstack-icon.svg" alt="BrowserStack" height="30"/></a> <a href="https://java.com"><img src="https://www.vectorlogo.zone/logos/java/java-icon.svg" alt="Java" height="30" /></a> <a href="https://www.selenium.dev/"><img src="https://seeklogo.com/images/S/selenium-logo-DB9103D7CF-seeklogo.com.png" alt="Selenium" height="30" /></a>
+
+Automate a real end-to-end user flow between app and browser in a single session. 
```

Comment:
Can we add little more description that this repo is particular for app automate platform

Reply:
Added proper description in README.